PapersCited.py

```diff
@@ -43,6 +43,8 @@ class PhrasesToChange:
       "suradnicima": "sur.",
       "suradnici": "sur.",
       "suradnika": "sur.",
+      # Change possesive form to a proper noun
+      "'s" : ""

```

Comment:
I hope something like "O’Sullivan, S.B., Schmitz, T.J. and Fulk, G.D. (2014) Physical Rehabilitation. 6th Edition" will not cause troubles 😄

Reply:
Good thinking. 
The replacement is case sensitive and only changes lowercase "s", so surnames such as O'Sullivan will be unaffected.
Will add a test to enforce it.

Reply:
Added test that confirms it in 7172274a